build.tmpl.yml

```diff
@@ -225,69 +222,62 @@ builds:
         - cmd: bash <(curl https://raw.githubusercontent.com/ory/xgoreleaser/master/rename.sh) "./dist/{{ .ProjectName }}_{{ .Version }}_{{ .Target }}.bom.json"
 
 archives:
-  - id: windows-cgo
```

Comment:
There was a reason for splitting this (i think install.sh/homebrew/docker/scoop had something to do with this), my suggestion is to not touch this and only change the replacing.

Reply:
reverted